grails-app/views/consentGroupReview/_index.gsp

```diff
@@ -0,0 +1,22 @@
+<html>
+  <head>
+    <meta name="layout" content="main_component">
+    <title>Consent Group</title>
+  </head>
+  <body>
+    <g:javascript>
+      const consentGroupReviewComponent = {
+        consentKey:'${issue.projectKey}',
+        consentGroupUrl: "${createLink(controller: 'consentGroup', action: 'getConsentGroup')}",
+        rejectConsentUrl: "${createLink(controller: 'newConsentGroup', action: 'delete')}",
+        projectKey: "${params.projectKey}",
+        updateConsentUrl: "${createLink(controller: 'newConsentGroup', action: 'update')}",
+        approveConsentGroupUrl: "${createLink(controller: 'newConsentGroup', action: 'approveConsentGroup')}"
```

Comment:
Pretty sure that these are already in `main_component` ... do you need to redefine them here too?
* consentKey
* createConsentGroupURL
* rejectConsentUrl
* projectKey
* updateConsentUrl
* approveConsentGroupUrl

Reply:
I wasn't paying attention to these, you're right! I've removed those from consentGroupReview Index, by previously check if they weren't in use, or replacing them with global constants.